Source/Initialization/InjectorDensity.H

```diff
@@ -65,14 +66,20 @@ struct InjectorDensityPredefined
         {
         case Profile::parabolic_channel:
         {
-            amrex::Real z_start   = p[0];
-            amrex::Real ramp_up   = p[1];
-            amrex::Real plateau   = p[2];
-            amrex::Real ramp_down = p[3];
-            amrex::Real rc        = p[4];
-            amrex::Real n0        = p[5];
-            amrex::Real n;
-            amrex::Real kp = PhysConst::q_e/PhysConst::c
+            // These are cast as double to ensure sufficient precision in the
+            // initialized profile density profile; without these, single and
+            // double precision versions of the executable can show disagreement
+            // From testing, it seems that (for at least some setups), it is only
+            // necessary to case n0 as double to get good agreement between single
+            // and double precision, but just in case, all are cast as double.
+            double z_start   = p[0];
+            double ramp_up   = p[1];
+            double plateau   = p[2];
+            double ramp_down = p[3];
+            double rc        = p[4];
+            double n0        = p[5];
+            double n;
+            double kp = PhysConst::q_e/PhysConst::c
                 *std::sqrt( n0/(PhysConst::m_e*PhysConst::ep0) );
```

Comment:
This line probably should use an amrex math function to be able to run on GPU (for the `sqrt`)

Reply:
Fixed.